packages/mobile/src/identity/commentEncryption.ts

```diff
@@ -0,0 +1,316 @@
+// Wallet utilities for comment encryption and comment key management
```

Comment:
The file is actually a rename of the `identity/commentKey.ts` but it's been mostly rewritten.
It handles comment key management tasks, encryption/decryption, and extracting identity metadata from comments